res/config/game_script/timetable_engine.lua

```diff
@@ -0,0 +1,115 @@
+local timetable = require "celmi/timetables/timetable"
+local timetableHelper = require "celmi/timetables/timetable_helper"
+
+local time = 0
+local clockstate = nil
+
+local timetableEngine = {}
+
+local co = nil
+local state = nil
+
+local timetableChanged = false
+
+local UIStrings = {

```

Comment:
Are `UIStrings` needed in both timetable_gui.lua and timetable_engine.lua?

Reply:
This is experimental. I wanted to remove any remnants of previous instances from the object. This way it is basically created from scratch.

Reply:
I must have overlooked something there.